storage/storage.go

```diff
@@ -0,0 +1,71 @@
+package storage
+
+import (
+	"strconv"
+	"sync"
+
+	"github.com/artemis13/platform-go-challenge/models"
+)
+
+var (
+	users = make(map[uint]*models.User)
+	mu    sync.RWMutex
+)
+
+// ConvertStringToUint safely converts a string to uint
+func ConvertStringToUint(s string) (uint, error) {
```

Comment:
The ConvertStringToUint is placed in the storage.go file and is being called from the handlers.go file. For the purpose of the project it would be more appropriate if it was placed in the handlers package in a helpers.go file containing various helper functions. As the project grows a utils package should be created containing all the helper functions.